src/sfr.cpp

```diff
@@ -155,43 +155,43 @@ namespace sfr {
     } // namespace mission
     namespace burnwire {
         // OP Codes 1900
-        SFRField<bool> fire = SFRField<bool>(false, 1900);
-        SFRField<bool> arm = SFRField<bool>(false, 1901);
-        SFRField<uint16_t> attempts = SFRField<uint16_t>(0, 1902);
-        SFRField<uint16_t> camera_attempts = SFRField<uint16_t>(0, 1904);
-        SFRField<uint32_t> start_time = SFRField<uint32_t>(0, 1903);
-        SFRField<uint32_t> burn_time = SFRField<uint32_t>(500, 1905);
-        SFRField<uint32_t> armed_time = SFRField<uint32_t>(2 * constants::time::one_day, 1906);
-        SFRField<uint16_t> mode = SFRField<uint16_t>((uint16_t)burnwire_mode_type::standby, 1907);
-        SFRField<uint16_t> attempts_limit = SFRField<uint16_t>(10, 1908);
+        SFRField<bool> fire = SFRField<bool>(false, 0x1900);
+        SFRField<bool> arm = SFRField<bool>(false, 0x1901);
+        SFRField<uint16_t> attempts = SFRField<uint16_t>(0, 0x1902);
+        SFRField<uint16_t> camera_attempts = SFRField<uint16_t>(0, 0x1904);
+        SFRField<uint32_t> start_time = SFRField<uint32_t>(0, 0x1903);
+        SFRField<uint32_t> burn_time = SFRField<uint32_t>(500, 0x1905);
+        SFRField<uint32_t> armed_time = SFRField<uint32_t>(2 * constants::time::one_day, 0x1906);
+        SFRField<uint16_t> mode = SFRField<uint16_t>((uint16_t)burnwire_mode_type::standby, 0x1907);
+        SFRField<uint16_t> attempts_limit = SFRField<uint16_t>(10, 0x1908);
     } // namespace burnwire
     namespace camera {
         // OP Codes 2000
-        SFRField<bool> photo_taken_sd_failed = SFRField<bool>(false, 2000);
-        SFRField<bool> take_photo = SFRField<bool>(false, 2001);
-        SFRField<bool> turn_on = SFRField<bool>(false, 2002);
-        SFRField<bool> turn_off = SFRField<bool>(false, 2003);
-        SFRField<bool> powered = SFRField<bool>(false, 2004);
+        SFRField<bool> photo_taken_sd_failed = SFRField<bool>(false, 0x2000);
```

Comment:
do we want to go 0x1A00 if we are doing hex - (that way its sequential still) 

also this comment is applicable for rest below this as well